app/routes/system_admin.py

```
@@ -858,315 +779,48 @@ def test_error_503():
-            active_invites.append(invite)
+def _resolve_report_id_from_ref(report_ref: str) -> int | None:
+    return resolve_opaque_ref('report', report_ref)
 
-    # Build rich admin data (from overview logic)
-    all_admins = User.query.filter(User.user_role == UserRole.TEACHER).order_by(User.id.asc()).all()
-    inactivity_threshold = utc_now() - timedelta(days=INACTIVITY_THRESHOLD_DAYS)
 
-    user_ids_by_admin_id = {
-        admin.id: admin.id
-        for admin in all_admins
+def _issue_to_view(issue):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_issue_to_view`

(ANN202)